lib/main.js

```diff
@@ -2,11 +2,11 @@
 /*globals module require*/
 var exec = require('child_process').exec
   , fs   = require("fs")
-  , path = require("path")
-  , util = require("util");
+  , path = require("path");
 
-function puts(error, stdout/*, stderr*/) {
-  util.puts(stdout);
+function consolelog(error, stdout, stderr) {
```

Comment:
I originally wanted to just name it `log`, but thought being verbose was safer. I'll change.